README.md

```diff
@@ -1,5 +1,5 @@
 [![Latest Release](https://img.shields.io/github/v/release/sda-se/sda-dropwizard-commons?label=latest)](https://github.com/SDA-SE/sda-dropwizard-commons/releases/latest)
```

Comment:
Everything in the readme should reference the new bundle (also the badge)

Reply:
I think the PR is not ready for review yet. I am actually trying to find all occurrences of `starter-swagger` in parallel..